### DISCUSSION TITLE
API key is not working as expected

### DISCUSSION THREAD

**Discussion by liuling73** [brianpetro/obsidian-smart-connections#421] [Q&A]
一直无法正常使用，点击测试API，反回结果：API key is not working as expected，API key是在官网获取的，是正确有效的KEY

**Comment by EvoAlg** [comment]
I get the same thing.

**Comment by brianpetro** [comment]
@EvoAlg @liuling73 check out https://github.com/brianpetro/obsidian-smart-connections/discussions/155#discussioncomment-7632050

**Comment by EvoAlg** [comment]
Thanks, I set up the second billing, deleted and old API key, then waited half an hour, then it worked.
